pdns/lua-recursor4.cc

```diff
@@ -384,7 +384,10 @@ RecursorLua4::RecursorLua4(const std::string& fname)
   d_lw->registerMember("type", &DNSRecord::d_type);
   d_lw->registerMember("ttl", &DNSRecord::d_ttl);
   d_lw->registerMember("place", &DNSRecord::d_place);
-  
+
+  d_lw->registerMember("content", &EDNSOptionView::content);
```

Comment:
This is the reason it is not NUL safe. 

If this were a `std::string`, Luawrapper would handle it in a NUL safe way. Here Luawrapper is not aware of the size field and interprets it as a NUL terminated string.

This could also cause issues with content without NULs in it, as it will continue to read until it encounters a NUL.